packages/console/src/cloud/pages/Main/index.tsx

```diff
@@ -18,8 +18,13 @@ function Protected() {
 
   useEffect(() => {
     const loadTenants = async () => {
-      const data = await api.get('/api/tenants').json<TenantInfo[]>();
-      setTenants(data);
+      try {
+        const data = await api.get('/api/tenants').json<TenantInfo[]>();
+        setTenants(data);
+      } catch {
+        // Note: this is intended to cause the app to crash.
+        throw new Error('Failed to load tenants.');
```

Comment:
The original error won't work?

Reply:
The original error may be an HTTP error, and we should crash the app in this case.
Other HTTP errors come from user interactions.